src/sailfishui/crash-reporter.qml

```diff
@@ -157,6 +157,32 @@ Page {
                 onClicked: PrivacySettings.allowMobileData = !PrivacySettings.allowMobileData
             }
 
+            SectionHeader {
```

Comment:
I'm not sure we should be having settings for this, let alone separating charger states. Developer tools can have more advanced settings but even then it's better if things just work and we can keep settings a little simpler. Assuming that 10-15% is enough for threshold, I'd think there is no-one saying 'but I want to drain my battery on 5% charge', or? :)

Reply:
Developer tools should be usable under any condition based on the developer's needs. So we have safe defaults but still allow to turn safety checks off.